src/test/java/com/box/sdk/BoxMetadataCascadePolicyTest.java

```diff
@@ -0,0 +1,144 @@
+package com.box.sdk;
+
+import com.eclipsesource.json.JsonObject;
+import com.github.tomakehurst.wiremock.client.WireMock;
+import com.github.tomakehurst.wiremock.junit.WireMockClassRule;
+import org.junit.Assert;
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+
+import java.io.IOException;
+import java.util.Iterator;
+
+/**
+ *
+ */
+public class BoxMetadataCascadePolicyTest {
+
+    /**
+     * Wiremock
+     */
+    @ClassRule
+    public static final WireMockClassRule WIRE_MOCK_CLASS_RULE = new WireMockClassRule(53621);
+    private BoxAPIConnection api = TestConfig.getAPIConnection();
+
+    @Test
+    @Category(UnitTest.class)
+    public void testCreateMetadataCascadePolicySucceedsSendsCorrectJson() throws IOException {
+        String result = "";
+        final String cascadePolicyURL = "/metadata_cascade_policies";
+        final String folderID = "22222";
+        final String scope = "enterprise_11111";
+        final String templateKey = "testTemplate";
+        JsonObject cascadeObject = new JsonObject()
+                .add("folder_id", folderID)
+                .add("scope", scope)
+                .add("templateKey", templateKey);
+
+        result = TestConfig.getFixture("BoxMetadataCascadePolicy/CreateMetadataCascadePolicies201");
+
+        WIRE_MOCK_CLASS_RULE.stubFor(WireMock.post(WireMock.urlPathEqualTo(cascadePolicyURL))
+                .withRequestBody(WireMock.equalToJson(cascadeObject.toString()))
+                .willReturn(WireMock.aResponse()
+                        .withHeader("Content-Type", "application/json")
+                        .withBody(result)));
+
+        BoxMetadataCascadePolicy.Info metadataCascadePolicyInfo = BoxMetadataCascadePolicy.create(this.api, folderID,
+                scope, templateKey);
+
+        Assert.assertEquals(folderID, metadataCascadePolicyInfo.getParent().getID());
+        Assert.assertEquals(scope, metadataCascadePolicyInfo.getScope());
+        Assert.assertEquals(templateKey, metadataCascadePolicyInfo.getTemplateKey());
+    }
+
+    @Test
+    @Category(UnitTest.class)
+    public void testGetAllMetadataCascadePoliciesSucceeds() throws IOException {
```

Comment:
After you add an instance method version of this call on `BoxFolder`, add a test for that method as well (should mostly be able to copy/paste this test)